_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -0,0 +1,246 @@
+
+```
+gcloud components install kubectl
+```
+
+Verify gcloud is configured with your login and the project you are using:
+```
+gcloud config list
+```
+
+If you do not wish to install the gcloud client, you can use  [Google Cloud Shell](https://cloud.google.com/shell/docs/){:target="_blank"} to perform most tasks.
+If you choose to use Cloud Shell, you can port forward and proxy using its [Web Preview](https://cloud.google.com/shell/docs/using-web-preview#previewing_the_application){:target="_blank"} feature.  For example, to access Grafana from Cloud Shell, change the kubectl port mapping from 3000:3000 to 8080:3000.  You can simultaneously preview four other consoles via Web Preview proxied on ranges 8080 to 8084.  
+
+> <img src="{{home}}/img/warning.svg" alt="Warning" title="Warning"
+ style="width: 32px; display:inline" />
+**NOTE:** You must set your default compute service account to include:
+
+> - ```roles/container.admin```  (Kubernetes Engine Admin)
+> - ```Editor```  (on by default)
+
+> To set this, navigate to the IAM section of the [Cloud Console](https://console.cloud.google.com/iam-admin/iam/project){:target="_blank"} and find your default GCE/GKE service account in the following form:
+
+```
+projectNumber-compute@developer.gserviceaccount.com
+```
+
+then under Roles, find the ```Kubernetes Engine``` group and set the role ```Kubernetes Engine Admin```.
+
+
+## Setup
+
+### Launch Deployment Manager
+
+Once you have an account and project enabled, proceed to the following link:
+
+- [Istio GKE Deployment
+  Manager](https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/GoogleCloudPlatform/deploymentmanager-samples/istio-example/examples/v2/gke/istio/cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/GoogleCloudPlatform/deploymentmanager-samples/istio-example/examples/v2/gke/istio/cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin){:target="_blank"}
+
+Its recommended to leave the the defaults on as the rest of the Quick Start demonstrates each feature.
+
+Click ```Deploy```
+
+![GKE-Istio Launcher](img/dm_launcher.png)
+
+> **NOTE:** Deployment can take upto 5 minutes.
+
+### Bootstrap gcloud
+
+Once Deployment completes, on your workstation with gcloud installed:
+
+- Bootstrap kubectl for the cluster you just created and confirm the cluster is
+  running and istio is enabled
+
+```
+gcloud container clusters list
+```
+```
+NAME           ZONE           MASTER_VERSION                    MASTER_IP       MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
+istio-cluster  us-central1-a  1.7.8-gke.0 ALPHA (29 days left)  130.211.216.64  n1-standard-2  1.7.8-gke.0   3          RUNNING
+```
+
+In this case, the cluster name is ```istio-cluster```
+
+Bootstrap to the cluster
+
+```
```

Comment:
Here and elsewhere, anything that represents something for the command-line should be annotated with ```bash

In case GitHub eats that, I mean three ticks and the word bash.

Reply:
done